ironic/tests/unit/drivers/modules/drac/test_management.py

```diff
@@ -423,17 +440,24 @@ def test_set_boot_device_called_with_unknown_boot_mode(
                           drac_mgmt.set_boot_device, self.node,
                           ironic.common.boot_devices.DISK, persistent=True)
 
+        mock_list_unfinished_jobs.assert_called_once_with(self.node)
         mock_validate_job_queue.assert_called_once_with(self.node)
         self.assertEqual(0, mock_client.change_boot_device_order.call_count)
         self.assertEqual(0, mock_client.set_bios_settings.call_count)
         self.assertEqual(0, mock_client.commit_pending_bios_changes.call_count)
 
     @mock.patch.object(drac_job, 'validate_job_queue', spec_set=True,
                        autospec=True)
+    @mock.patch.object(drac_job, 'list_unfinished_jobs', spec_set=True,
+                       autospec=True)
     def test_set_boot_device_with_invalid_job_queue(
-            self, mock_validate_job_queue, mock_get_drac_client):
+            self, mock_list_unfinished_jobs, mock_validate_job_queue,
+            mock_get_drac_client):
         mock_client = mock.Mock()
         mock_get_drac_client.return_value = mock_client
+
+        mock_list_unfinished_jobs.side_effect = exception.DracOperationError(
```

Comment:
I think in this test, we don't want list_unfinished_jobs to raise an exception, so I think the line above and below should be removed.

Reply:
Let's add another test case that's pretty much the same as this one that tests list_unfinished_jobs raising an exception.

Reply:
Done